docs/topics/clusterdefinitions.md

```diff
@@ -910,6 +910,8 @@ https://{keyvaultname}.vault.azure.net:443/secrets/{secretName}/{version}
 | enableAutomaticUpdates        | no       | If set to `true` Windows Update will be configured to automatically apply updates on Windows nodes. Default: `false`                                                                                                                                            |
 | enableCSIProxy                | no       | If set to `true` the csi-proxy specified by `windowsProfile.csiProxyURL` will get installed during node provisioning. See [Windows Csi Proxy](csi-proxy-windows.md) for more details.                                                                           |
 | provisioningScriptsPackageURL | no       | Path to a package containing provisioning scripts found in [/staging/provisioning/windows](/staging/provisioning/windows). Default values will be maintained by aks-engine team and contain script content signed by Microsoft.                                 |
+| windowsPauseImageURL          | no       | Path to a Windows pause image. Default values will be maintained by aks-engine team.                                 |
```

Comment:
can we add an example to the docs:  # e.g. "mcr.microsoft.com/oss/kubernetes/pause:1.4.0"

```suggestion
| windowsPauseImageURL          | no       | Url to a Windows pause image. Default values will be maintained by aks-engine team.  e.g. "mcr.microsoft.com/oss/kubernetes/pause:1.4.0"                                |
```